pkg/accounting/tally/tally.go

```diff
@@ -69,14 +69,22 @@ func (t *Tally) Run(ctx context.Context) (err error) {
 //Tally calculates data-at-rest and bandwidth usage once
 func (t *Tally) Tally(ctx context.Context) error {
 	//data at rest
```

Comment:
nit: space between // and text for consistency

Reply:
throughout the code base for comments some places have space after `//` and some places do not.  It seems like in this file most/all of the comments do not have the extra space. I would argue to keep them all that way instead of having to change them all.  In a prefect world we would have consistency across the entire repo, but that seems unrealistic and already failed. In my opinion, consistency in a single file/package is the best we can do.